localization/english/load_tips/load_tips_l_english.yml

```diff
@@ -0,0 +1,114 @@
+ LOADING_TIP_11:0 "“Pandaria, her hills of gold\nIn dark and mournful times of old\nDid once a hopeless horror hold\nWhen from her sacred veil did spring\nWith storm and flash, a monstrous thing\nHis name Lei Shen, the Thunder King...”\n #bold —Lorewalker Cho#!"
+ LOADING_TIP_12:0 "Humans, night elves, dwarves, gnomes, draenei, and the savage worgen make up the illustrious Alliance. Proud and noble, courageous and wise, these races work together to preserve order in Azeroth. The Alliance is driven by honor and tradition. Its rulers are champions of justice, hope, knowledge, and faith.\n\nIn a time when chaos and uncertainty reign, the Alliance remains steadfast in its determination to bring light to the darkest corners of the world."
+ LOADING_TIP_13:0 "The Horde is made up of orcs, forsaken, tauren, trolls, blood elves, and most recently, goblins. Misunderstood and cast aside, these diverse and powerful races strive to overcome their differences and unite as one in order to win freedom for their people and prosper in a land that has come to hate them.\n\nIn the Horde, action and strength are valued above diplomacy, and its leaders earn respect by the blade, wasting no time with politics. The brutality of the Horde's champions is focused, giving a voice to those who fight for survival"
+ LOADING_TIP_14:0 "#T Did you know?#!\n\nGul'dan was banished from his village because of his disability. He later slaughtered everyone in the village, including those who showed him kindness."
+ LOADING_TIP_15:0 "“The beginning of wisdom is the statement 'I do not know.' The person who cannot make that statement is one who will never learn anything. And I have prided myself on my ability to learn.”\n #bold —Thrall#!"
+ LOADING_TIP_16:0 "“Know this: a shaman must never demand power; to do so would arouse the anger of the elements, and a shaman without the elements' protection is nothing. Those that follow the path of the shaman must never forget this most vital tenet. However, a shaman that works in a respectful union with the elements wields a power rivaled by few.”\n #bold —Teo Hammerstorm#!"
+ LOADING_TIP_17:0 "“Lost time is never found again. Oh, never mind I found it.”\n #bold —Chronormu (Chromie)#!"
+ LOADING_TIP_18:0 "#T Did you know?#!\n\nAside from her wisdom and beauty, Alexstrasza can be recognized by the gems on her wrists, the 2nd digit of her left paw, and her huge horns. She also has an opal chain on her chin."
+ LOADING_TIP_19:0 "“Pain... Agony... My hatred burns through the cavernous deeps. The world heaves with my torment. Its wretched kingdoms quake beneath my rage. But at last... The whole of Azeroth will break... And all will burn beneath the shadow of my wings.”\n #bold —Neltharion#!"
+ LOADING_TIP_20:0 "#T Did you know?#!\n\nDeathwing can easily be recognized by his mandible, and the colossal blade on the end of his gigantic tail."
```

Comment:
LOADING_TIP_20:0 "#T Did you know?#!\n\nDeathwing can easily be recognized by his mandible and the colossal blade on the end of his gigantic tail."